src/inverseRobot.js

```diff
@@ -25,6 +25,23 @@
  */
 function inverseRobot(robot) {
   // write code here
+  const output = robot;
```

Comment:
you don't create a copy, but just make a reference to same object `robot` - does not provide any value.